spec/rubocop/cop/rspec/expect_in_hook_spec.rb

```diff
@@ -76,4 +76,56 @@
       end
     RUBY
   end
+
+  context 'with config IgnoreSharedGroups set to false' do
+    let(:cop_config) { { 'IgnoreSharedGroups' => false } }
+
+    it 'does not accepts `expect` in `shared_examples`' do
+      expect_offense(<<-RUBY)
+      shared_examples 'for shared setup' do
+        before do
+          expect(object).to receive(:message)
+          ^^^^^^ Do not use `expect` in `before` hook
+          expect_any_instance_of(something).to receive(:foo)
+          ^^^^^^^^^^^^^^^^^^^^^^ Do not use `expect_any_instance_of` in `before` hook
+        end
+      end
+        expect(object).to receive(:message)
+        expect_any_instance_of(something).to receive(:foo)
+      RUBY
+    end
+  end
+
+  context 'with config IgnoreSharedGroups set to true' do
+    let(:cop_config) { { 'IgnoreSharedGroups' => true } }
```

Comment:
Ok, well. We have those two examples below. What makes you think both of them should be ignored by the cop?